### DISCUSSION TITLE
SQLCipher iOS 15.6

### DISCUSSION THREAD

**Discussion by Foxsterc** [simolus3/drift#2683] [Q&A] [answered]
Interesting issue that affects a released app on iOS 15.6.

Install the app from the store, app installs fine works fine. Close the app down and re-open it and i fails! Selecting the pragma ciper_version is empty on restart?

Any ideas?

**Comment by simolus3** [comment, marked as answer]
> Selecting the pragma ciper_version is empty on restart?

But `cipher_version` returned something on the first app start? Did you check [this section](https://pub.dev/packages/sqlcipher_flutter_libs#incompatibilities-with-sqlite3-on-ios-and-macos) to make sure you're not accidentally linking the normal sqlite3 as well?

**Reply by Foxsterc** [reply]
Yes, that was it I used the pod file fix to remove it. Thank you
